### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.4.11 Non-text Contrast

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#655]
(Mary Jo) 

- No comment for now...We will have to follow the MATF continuing discussions on Issue 49. 

(Chris L) 

- This discussion on issue 49 appears to be more about what the vendors would do and what they would report on as fails vs. how it would apply in terms of user agents discussion. 
- A lot of this discussion leans on pass/fails of evaluating vs. whether or not in applies to native mobile. 
- Discussion on what is native mobile and what is user agent is an ongoing discussion within the discussion thread. 
- Do we as a group want to subscribe to all of these discussions to monitor or rely on WCAG2Mobile to come to WCAG2ICT with clarity per their reviews?

References:
- https://w3c.github.io/matf/#success-criterion-1-4-11-non-text-contrast
- https://github.com/w3c/matf/issues/49
